libavcodec/vvc/vvc_ctu.c

```diff
@@ -865,8 +865,12 @@ static enum IntraPredMode derive_center_luma_intra_pred_mode(const VVCFrameConte
     const int cu_pred_mode        = SAMPLE_CTB(fc->tab.cpm[0], x_center, y_center);
     const int intra_pred_mode_y   = SAMPLE_CTB(fc->tab.ipm, x_center, y_center);
 
-    if (intra_mip_flag)
-        return INTRA_PLANAR;
+    if (intra_mip_flag) {
+        if (cu->tree_type == SINGLE_TREE && sps->chroma_format_idc == CHROMA_FORMAT_444)
+            return INTRA_DEFAULT;
+        else
```

Comment:
else is not needed, you can return INTRA_PLANAR directly. 
thank you

Reply:
Done in 6e7b9747700a06ba1427f74aa0050ec70e243338